app/lib/linters/crunch_api.rb

```diff
@@ -0,0 +1,97 @@
+require 'rest-client'
+require 'base64'
+module Linters
+
+  # A class called Crunch
+  class CrunchApi
+
+    attr_accessor :base_url
+
+    def initialize()
+      @base_url = "https://platform.42crunch.com"
+    end
```

Comment:
Base URL looks like a constant to me. Therefore, I'd suggest a better way to handle this would be to use:
```
BASE_URL = "https://platform.42crunch.com"